src/Orleans.Runtime/Silo/Silo.cs

```diff
@@ -361,8 +361,11 @@ await scheduler.QueueAction(catalog.Start, catalog.SchedulingContext)
 
             // SystemTarget for provider init calls
             this.fallbackScheduler = Services.GetRequiredService<FallbackSystemTarget>();
-
             RegisterSystemTarget(fallbackScheduler);
+
+            // SystemTarget for startup tasks
+            var startupTaskTarget = Services.GetRequiredService<StartupTaskSystemTarget>();
```

Comment:
Not sure what we need to setup another task scheduler, as we already have and use fallback for this.  It's not a bad thing, just not convinced it's necessary.

Reply:
It's not necessary, but during our meeting it was mentioned, so I added a new special-purpose one. We can always axe it, especially when we fix scheduling of lifecycle participants more broadly